xtext-gradle-builder/src/main/java/org/xtext/gradle/builder/GradleValidatonCallback.xtend

```diff
@@ -14,6 +14,7 @@ class GradleValidatonCallback implements IPostValidationCallback {
 	boolean errorFree = true
 
 	override afterValidate(URI validated, Iterable<Issue> issues) {
+		logger.info("Starting validation for input: '" + validated.lastSegment + "'")

```

Comment:
❌ Now we have it twice for Xtext < 2.15

Reply:
i wonder if we somehow can detect the version here